config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,13 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (c Config) Validate() (r report.Report) {
+	for _, fs := range c.Storage.Filesystems {
+		fsPath := *fs.Path
+		if len(fsPath) < 4 || (fsPath[:4] != "/etc" && fsPath[:4] != "/var") {
+			r.AddOnError(path.New("filesystems", 0), common.ErrMountPointForbidden)
```

Comment:
hmm, we should report correct number here.

Reply:
not sure why i didn't catch that my bad!